test/linear_nonlinear_krylov_tests.jl

```diff
@@ -49,8 +49,8 @@ end
     prob = ODEProblem(f, u0, (0.0, 1.0))
     prob_ip = ODEProblem{true}(f_ip, u0, (0.0, 1.0))
 
-    dt = 0.1; tol=1e-5
-    Algs = [Exp4]
+    dt = 0.05; tol=1e-5

```

Comment:
was there an issue here?

Reply:
If you recall our numerical experiments on `phiv_timestep!`, the provided tolerance is more of a general guideline than a hard limit. This is especially true for small time steps as there's not much room of improvement for the Krylov part and most of the error comes directly from the truncation error of the schemes. 

Using the previous `dt` would fail the test (by a small margin), so I turned it down a bit.

Now that you mentioned it, comparing the results to Tsit5 is not a good test and should probably be rewritten using `test_convergence`. It's best if we can find a nonlinear system with known solution which we can test on.

Reply:
Oh I just meant the `dt` change.

We don't need analytical solutions for convergence testing, though we should probably switch to using a better analytical solution when https://github.com/JuliaDiffEq/DiffEqProblemLibrary.jl/issues/8 is there. @Vaibhavdixit02 did you ever add the analytical solution? https://github.com/JuliaDiffEq/DiffEqSensitivity.jl/pull/14